### PR TITLE
Add central publish workflow

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
To manually publish artefacts to the Ballerina central.

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: .github/workflows/central-publish.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        required: true
+
+  jobs:
+    publish-release:
+      runs-on: ubuntu-latest
+      if: github.repository_owner == 'ballerina-platform'
+      steps:
+        -   uses: actions/checkout@v2
+        -   name: Checkout Tag
+            run: git checkout v${{ github.event.inputs.ballerinaVersion }}
```
Comment (BuddhiWathsala) [inline .github/workflows/central-publish.yml:16]: Anyway, we have to create this branch manually right?

File: .github/workflows/central-publish.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        -   name: Grant execute permission for gradlew
+            run: chmod +x gradlew
+        -   name: Publish artifact
+            env:
+              BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_ACCESS_TOKEN }}
+              packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+              packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+              GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+            run: |
+              ./gradlew ballerinaPublish
```
Comment (BuddhiWathsala) [inline .github/workflows/central-publish.yml:30]:
```suggestion
              ./gradlew ballerinaPublish
              
```

File: .github/workflows/central-publish.yml (hunk 1 above)
Comment (niveathika) [reply .github/workflows/central-publish.yml:16]: No this will checkout the release tag

Comment (niveathika) [reply .github/workflows/central-publish.yml]: Ah the command should be changed to 
git checkout v${{ github.event.inputs.releaseVersion }}

**Pull request closed without merging**